server/quizzes/controller.js

```diff
@@ -76,3 +80,45 @@ export const rateQuiz = async (body, user) => {
     throw newError(StatusCode.Error, error.message);
   }
 };
+
+// Helper to score quiz on back-end before POSTing
+const scoreQuiz = async (quizID, userAnswers) => {
+  const score = await getQuiz(quizID).then((res) => {
```

Comment:
I've changed `getQuiz` to return the `quiz` object instead of the firestorm quiz